fairworkflows/fairstep.py

```diff
@@ -85,9 +86,12 @@ class FairStep(RdfWrapper):
 
     def __init__(self, label: str = None, description: str = None, uri=None,
                  is_pplan_step: bool = True, is_manual_task: bool = None,
-                 is_script_task: bool = None, inputs: List[FairVariable] = None,
+                 is_script_task: bool = None,
+                 language: LinguisticSystem = LINGSYS_ENGLISH,
```

Comment:
Yes, should we think about renaming it to ```FairBase``` or something now? Or perhaps I should make a ```FairBase``` that inherits from ```RdfWrapper```, since we probably intend to make ```FairRetro``` use ```RdfWrapper``` but the language and script task stuff wouldn't make sense for it...?

Reply:
Hmmm, actually, I guess the ```FairRetro``` output in some sense could have a language? Its description/label is most likely to be in a natural language (probably English), and if not it doesn't _need_ to be provided, so we can leave it as `None` if we really don't want to set a language. OK, I'll move this stuff.

Reply:
Note that this still needs to be in the `FairStep` or `FairWorkflow` constructor signature though, since that's how it gets passed to the `RdfWrapper` superclass. So the only thing I'll move is the following:
```
        if language is not None:
            self.language = language
```

Reply:
In which case... shouldn't we also move `description`, `label` etc too?

Reply:
OK, I've moved `description` and `label` to `RdfWrapper` too now, because they were identical in `FairStep` and `FairWorkflow`.